### PR TITLE
Add option to run redis based import workers

### PR DESCRIPTION
**Pull request by janLo**
This adds an option to run the redis based import workers.

### CODE REVIEW

**Comment by xsteadfastx** [comment]
i think thats a really good idea. but i never used the importet so some wallabag dev should check if that is all right.

File: README.md [hunk 1]
```diff
@@ -136,3 +136,11 @@ server {
 
 }
 ```
+
+## Importer worker
+
+To run the [async redis importer worker](http://doc.wallabag.org/en/master/developer/asynchronous.html#install-redis-for-asynchronous-tasks) use the following command:
+```
+$ docker run --name wallabag --link wallabag-db:wallabag-db --link redis:redis -e <... your config variables here ...>  wallabag/wallabag importer <type>
```
Comment (j0k3r) [inline README.md]: Shouldn't this be `import` instead of `importer`?

Comment (janLo) [inline README.md]: You're right. Will fix it as soon as I find time.

**Comment by janLo** [comment]
Fixed and rebased on master.

**Comment by j0k3r** [comment]
Thanks !
I've fixed few typos.

**Pull request merged**